src/Ztm.Zcoin.Rpc.Tests/ZcoinRpcClientTests.cs

```diff
@@ -712,6 +714,92 @@ public async Task GetExodusPayloadAsync_WithValidExistTransaction_ShouldSuccess(
             );
         }
 
+        [Fact]
+        public async Task GetBlockAsync_WhichContiainExodusTransaction_ShouldSuccess()
+        {
+            // Arrange.
+            var issuer = new PropertyIssuer(this.node, this.subject);
+            this.node.Generate(101);
+
+            var property = await issuer.IssueManagedAsync();
+            await property.GrantAsync(new PropertyAmount(1000));
+
+            var destination = await this.subject.GetNewAddressAsync(CancellationToken.None);
+            var rawTx = await this.subject.SendTokenAsync(property.Owner, destination, property.Property, new PropertyAmount(10), null, null, CancellationToken.None);
+
+            await this.subject.SendRawTransactionAsync(rawTx, CancellationToken.None);
+            var id = this.node.Generate(1).First();
+
+            var exodusTransaction = new TestExodusTransaction(TestAddress.Regtest1, TestAddress.Regtest2);
+            this.encoder.Decode(Arg.Any<BitcoinAddress>(), Arg.Any<BitcoinAddress>(), Arg.Any<byte[]>())
```

Comment:
Better to match with sender and receiver so at least we know it is actually what we expect.